R/ggplotly.R

```diff
@@ -176,8 +176,16 @@ gg2list <- function(p){
     df <- df[order(df$order),]
     df$order <- NULL
 
-    misc$prestats.data <- merge(built$prestats.data[[i]],
-                                gglayout[, c("PANEL", "plotly.row", "COL")])
+    prestats <- built$prestats.data[[i]]
+    ## scale_reverse multiples x/y data by -1, so here we undo that so
```

Comment:
Please use a single `#` for comments (see https://github.com/ropensci/plotly/wiki/Development-guidelines#coding-style and reference therein https://google-styleguide.googlecode.com/svn/trunk/Rguide.xml#comments).